spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskOperations.java

```diff
@@ -138,6 +138,14 @@ public interface TaskOperations {
 	 */
 	String taskExecutionLog(String externalExecutionId, String platform);
 
+	/**
+	 * Return the task execution log.
+	 *
+	 * @param executionId the task execution identifier.
```

Comment:
It might be helpful to add "internal" and/or compare/contrast to the other "external get logs" api. IOW - when do I use one or the other?